test/utils.py

```diff
@@ -0,0 +1,51 @@
+from collections.abc import Sequence
```

Comment:
Would make sense to move this to the library code, right?

Reply:
Moved it to `tianshou/utils/space_info.py`